ui/src/id.ts

```diff
@@ -0,0 +1,10 @@
+import * as org from "./org";
+import * as user from "./user";
+
+// Check if the given id is already taken
+export const isTaken = (id: string): Promise<boolean> =>
+  org.getOrg(id).then(org => !!org || user.get(id).then(user => !!user));
```

Comment:
@xla fyi, first I did:

```
org.getOrg(id).then(org => !!org)  || user.get(id).then(user => !!user);
```

While it worked, the pre-commits complained that I was using a promise in a bool condition, which made sense.

Reply:
You should chain it with `then` calls. Happy to pair on that.

Reply:
Would be nice :) I'd expect what I tried and what I have to be "chain it with 'then' calls", look forward to pairing. thanks